### ISSUE TITLE
'any' JSON Schema type should be mapped to a json::Value instead of a String

### ISSUE THREAD

**Issue by xDarksome** [Byron/google-apis-rs#403]
Related to #336

Currently there is a `TODO`  stating:
```
# TODO: Figure out how to handle it. It's 'interface' in Go ...
````
(https://github.com/Byron/google-apis-rs/blob/main/src/generator/lib/types.py#L33)

I think the only suitable type here is `json::Value`.    
Using `String`  makes it it impossible to build some payloads, as mentioned in #336.

I would send a PR myself, but, for some reason, after regenerating the code I am getting a diff with size of a half of the project :smiling_face_with_tear:  And the changes are mostly unrelated.

Anyway, changing `String` to `json::Value` and regenerating the code should be sufficient.

**Comment by Byron** [comment]
That's awesome! Please feel free to submit a PR, without regenerating all the APIs. 

Locally you should be able to test your changes with commands like `make drive3-cli-cargo drive3-cargo ARGS=check`, substituting the CLI or API that you are interested in.

**Comment by tyranron** [comment]
@Byron I guess this may be closed now, as should be fixed by #404.

**Comment by Byron** [comment]
Thanks for the triage-support. It's appreciated.
